src/main/java/com/snoop/controllers/WorkspaceController.java

```diff
@@ -59,6 +59,10 @@ public String index(Map<String, Object> model) {
 
 		Workspace wkps = new Workspace(new Long(1));
 		
+		if(wkps.getId().equals(null + "")){
+			//TODO: do something!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)